tensorboard/plugins/hparams/_keras_test.py

```diff
@@ -46,12 +38,12 @@ def _initialize_model(self, writer):
             "optimizer": "adam",
             HP_DENSE_NEURONS: 8,
         }
-        self.model = keras.models.Sequential(
+        self.model = tf.keras.models.Sequential(
```

Comment:
Just curious but I haven't tested this myself - does this need to be changed to `tf.keras.Sequential `: https://keras.io/api/models/sequential/

Reply:
Looking at the source tf.keras.model.Sequential and tf.keras.Sequential appear to point to the same thing.